code/game/objects/items/weapons/mine.dm

```diff
@@ -1,43 +1,58 @@
 /obj/item/weapon/mine
 	name = "Excelsior Mine"
-	desc = "For the excelsior! Just add igniter with signaler or sensor in it, and it's ready to blow."
+	desc = "For the excelsior! A mercifully brief end to anyone who steps on it. Will ignore Excelsior Comrades"
```

Comment:
"An anti-personnel mine. IFF technology grants safe passage to Excelsior agents, and a mercifully brief end to others."